sickbeard/tv.py

```diff
@@ -1439,7 +1439,10 @@ def downloadSubtitles(self, force=False):
                         helpers.chmodAsParent(subtitle.path)
 
```

Comment:
except ServiceError as e:
            logger.log("Service is unavailable: {0}".format(str(e)), logger.INFO)
        except Exception:
            logger.log("Error occurred when downloading subtitles: " + traceback.format_exc(), logger.DEBUG)